OracleTuxedo/core/samples/shm/Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Pull Tuxedo base image
-FROM oracle/tuxedo:<VERSION>
+FROM oracle/tuxedo:12.2.2
 
 MAINTAINER Todd Little <todd.little@oracle.com>
 COPY simpapp_runme.sh /home/oracle/
```

Comment:
These should be copied to somewhere outside the home directory. I would suggest `/u01` to match the WebLogic stuff.